science/paraview/Portfile

```diff
@@ -118,12 +114,25 @@ pre-configure {
         -DCMAKE_C_COMPILER=${configure.cc}
 
        if {[mpi_variant_isset]} {
+           depends_lib-append port:icet
+           mpi.enforce_variant icet
+
            configure.args-append  \
                -DPARAVIEW_USE_MPI:BOOL=ON \
                -DMPI_C_COMPILER=${mpi.cc} \
                -DMPI_CXX_COMPILER=${mpi.cxx} \
                -DMPIEXEC=${mpi.exec} \
-               -DMPI_Fortran_COMPILER=${mpi.fc}
+               -DMPI_Fortran_COMPILER=${mpi.fc} \
+               -DVTK_USE_SYSTEM_DIY2:BOOL=OFF
+
+            if {${os.major} == 10 && ${os.minor} == 12} {
```

Comment:
This should be `if {${os.major} == 16} {`. See https://trac.macports.org/wiki/PortfileRecipes